packages/block-library/src/rss/edit.js

```diff
@@ -119,6 +119,8 @@ class RSSEdit extends Component {
 							onChange={ ( value ) => setAttributes( { itemsToShow: value } ) }
 							min={ DEFAULT_MIN_ITEMS }
 							max={ DEFAULT_MAX_ITEMS }
+							allowReset={ true }
+							initialValue={ 5 }
```

Comment:
Are these changes meant to be included here? I don't see the relation to the original issue #11756.

Reply:
whoops! you're right. This was for another issue that I was fixing.